pkg/syncer/metadatasyncer.go

```diff
@@ -551,6 +573,217 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 	return nil
 }
 
+// startTopologyCRInformer creates and starts an informer for CSINodeTopology custom resource.
+func startTopologyCRInformer(ctx context.Context, cfg *restclient.Config) error {
```

Comment:
Are you making sure you are calling `startTopologyCRInformer` only for Vanilla flavor?

Reply:
Yes, verified again